src/assets/i18n/en.json

```diff
@@ -556,61 +556,61 @@
         "BLOG_HEADING": "What's New",
         "VIEW_BLOG_LINK": "Read more on the blog"
       },
-      "COLLECTIONS": [{
-          "SUBHEADING": "Copyright in Artstor",
-          "DESCRIPTION": "Learn more with our guide",
-          "CAPTION": "Artstor LibGuides",
-          "LINK": "http://artstor.libguides.com/Copyright-in-Artstor/",
-          "IMG_SOURCE": "/assets/img/featured-collections/Copyright_LibGuide.jpg",
-          "LINK_TITLE": "Copyright in Artstor",
-          "ALT_TEXT": "Copyright in Artstor"
+      "COLLECTIONS": [
+        {
+        "SUBHEADING": "The art of medicine",
+        "DESCRIPTION": "Eakins’ medical paintings and clinical fact",
+        "CAPTION": "Thomas Eakins. The Gross Clinic, detail. 1875. Image and data provided by University of Georgia Libraries",
+        "LINK": "https://www.artstor.org/2019/12/04/the-drama-of-the-operating-theater-thomas-eakins-medical-paintings-and-clinical-fact/",
+        "IMG_SOURCE": "/assets/img/featured-collections/Carnegie_Eakins.jpg",
+        "LINK_TITLE": "Eakins’ medical paintings and clinical fact",
+        "ALT_TEXT": "Eakins’ medical paintings and clinical fact"
```

Comment:
nit - indents here might be a smidge off with the `{` at the same level as its contents.